android-json-form-wizard/src/test/java/com/vijay/jsonwizard/utils/NativeFormLangUtilsTest.java

```diff
@@ -118,4 +120,12 @@ public void testResourceBundleWithPropertiesFromDbIsEmptyWhenClientFormDoesntExi
         ResourceBundle mlsResourceBundle = NativeFormLangUtils.getResourceBundleFromRepository(RuntimeEnvironment.application, interpolatedJsonForm);
         assertFalse(mlsResourceBundle.getKeys().hasMoreElements());
     }
+
+    @Test
+    public void testPerformFormsTranslationShouldReturnCorrectValue() {
+        NativeFormLibrary.getInstance().setPerformFormTranslation(true);
+        assertTrue(NativeFormLibrary.getInstance().isPerformFormTranslation());
+        NativeFormLibrary.getInstance().setPerformFormTranslation(false);
+        assertFalse(NativeFormLibrary.getInstance().isPerformFormTranslation());
+    }
```

Comment:
This test is not too relevant now, it can be removed.

Reply:
Test cases removed